src/rules-engine/triggers/EventTrigger.js

```diff
@@ -36,6 +38,10 @@ class EventTrigger extends Trigger {
 
   async start() {
     await this.thingConn.start();
+    await this.subscribe();
+  }
+
+  async subscribe() {
```

Comment:
`this.subscribed` should probably be set in this function.

Reply:
Setting this.subscribed in this function would either have no effect (`this.subscribed = false`, it's already false) or be incorrect (`this.subscribed = true`, the Thing model hasn't acknowledged the subscription request)

If there were a way for the Thing model to drop a subscription and require a transition from subscribed to unsubscribed then this would be a different story.